doc/modules/preprocessing.rst

```diff
@@ -426,6 +426,47 @@ features, one has to explicitly set ``n_values``. For example,
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as integers.
 
+.. _discretization:
```

Comment:
Here's the rendered version:

<img width="841" alt="screen shot 2017-02-26 at 3 43 25 am" src="https://cloud.githubusercontent.com/assets/1865885/23339375/ce2d93b6-fbd5-11e6-9ccd-3d0749bffb54.png">

I think my narrative is a little weak. Are there suggestions on how to improve it?